projects/storefrontlib/src/cms-components/configurator/commons/config-form/config-form.component.html

```diff
@@ -24,21 +24,21 @@
               *ngIf="attribute.uiType === UiType.RADIOBUTTON"
               [attribute]="attribute"
               [group]="group.id"
-              [productCode]="configuration.productCode"
+              [productCode]="configuration.owner.key"
               (selectionChange)="updateConfiguration($event)"
             ></cx-config-attribute-radio-button>
             <cx-config-attribute-drop-down
               *ngIf="attribute.uiType === UiType.DROPDOWN"
               [attribute]="attribute"
               [group]="group.id"
-              [productCode]="configuration.productCode"
+              [productCode]="configuration.owner.key"
               (selectionChange)="updateConfiguration($event)"
             ></cx-config-attribute-drop-down>
             <cx-config-attribute-input-field
               *ngIf="attribute.uiType === UiType.STRING"
               [attribute]="attribute"
               [group]="group.id"
-              [productCode]="configuration.productCode"
+              [productCode]="configuration.owner.key"
```

Comment:
Should be called ownerKey instead of productCode

Reply:
Done